CHANGELOG.rst

```diff
@@ -49,6 +49,7 @@ Deprecations and removed features
 - ``set --query`` now returns a falsy status of 255 if given no variable names. This means ``if set -q $foo`` will not enter the if-block if ``$foo`` is empty or unset. To restore the previous behavior you would use something like ``if not set -q foo; or set -q $foo``. We do not expect anyone to have used this on purpose, any places this happens are almost certainly buggy (:issue:`8214`).
 - Mac OS X 10.9 is no longer supported. The minimum Mac version is now 10.10 "Yosemite."
 - ``_`` is now a reserved keyword (:issue:`8342`).
+- The special input functions ``nextd-or-forward-word``, ``prevd-or-backward-word``, and ``delete-or-exit`` (for readline functionality in bindings) are no longer available as normal fish functions.
```

Comment:
I guess to be precise

	New special input functions ``nextd-or-forward-word``, ``prevd-or-backward-word`` replace fish functions of the same name.
	Special input function ``delete-or-exit`` is no longer available as fish function.

Which makes me realize that this is a breaking change for whoever added custom bindings for `nextd-or-forward-word`.
I don't know if that's common (maybe try a quick github search). It might be better to keep a `function nextd-or-forward-word; commandline -f nextd-or-forward-word; end`

Reply:
What do you mean by custom bindings?

Reply:
Oops I somehow thought this breaks `bind X nextd-or-forward-word` but it only breaks `bind X 'nextd-or-forward-word; echo something else'`. So the risk of breakage seems very low.

Reply:
I did have two other thoughts:

1. How much deliberation was there over the name "special input function"? IMHO it's confusing that "special input functions" are not actually (fish) functions -- what do you think about calling them "binding actions" or something else that doesn't have the word function in it?
2. AFAIK there is no way to invoke these things except in a binding (correct me if I'm wrong) -- what do people think about adding a `fish_special_input_function` (or `fish_binding_action`) command which can be used to invoke `fish_binding_action nextd-or-forward-word`, for the benefit of those few people who do somehow use these "functions" in custom bindings?

Reply:
`commandline -f` allows you to invoke input functions.

Reply:
Ah, thanks.

Ok, the copy edits should be resolved.